.github/workflows/kubectl-deploy-job-eks.yml

```diff
@@ -0,0 +1,91 @@
+name: Superlinter
+
+on:
+  workflow_call:
+    secrets:
+      token:
+        description: 'Github token'
+        required: true
+      aws_access_key_id:
+        description: 'AWS_ACCESS_KEY_ID'
+        required: true
+      aws_secret_access_key:
+        description: 'AWS_SECRET_ACCESS_KEY'
+        required: true
+      aws_default_region:
+        description: 'AWS_DEFAULT_REGION'
+        required: true
+      clustername:
+        description: 'Cluster Name'
+        required: true
+
+    inputs:
+      registry:
+        description: 'registry for the docker images'
+        default: 'ghcr.io'
+        required: false
+        type: string
+      registry_path:
+        description: 'registry path for the repo'
+        required: true
+        type: string
+      job_name:
+        description: 'name of job'
+        required: true
+        type: string
+      job_file:
+        description: 'filename of the job'
+        default: 'job.yaml'
+        required: true
+        type: string
+      tag:
+        description: 'Image tag to deploy'
+        default: 'latest'
+        required: true
```

Comment:
```suggestion
        required: false
```
Same comment

Reply:
Going to remove the default on this one.